packages/components/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@web3-ui/components",
+  "license": "MIT",
+  "version": "1.0.0",
+  "private": false,
+  "engines": {
+    "node": ">=16.0.0",
+    "yarn": "^1.5"
+  },
+  "description": "React Typescript Library Boilerplate",
+  "author": "Dhaiwat Pandya",
+  "homepage": "https://github.com/Developer-DAO/web3-ui/",
+  "bugs": {
+    "url": "https://github.com/Developer-DAO/web3-ui/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Developer-DAO/web3-ui/"
+  },
+  "scripts": {
```

Comment:
maybe a future question, but do we want these scripts to live at the package level or the root level? I'd be a bit worried that the scripts for each package might diverge, and i'd rather us end up with something like `yarn test packages/components` at the root instead of `cd packages/components && yarn test` or `yarn workspace @web3-ui/components test`. food for thought

Reply:
I wanted to raise a similar discussion. Chakra has everything in the root folder, so the test and jest config are at the root and the packages only have the test file. So at the root we'd have `yarn test` and this would run all tests for example.

My comment in the PR description "Test setup in the root folder (and maybe other root configs too)", was referring to those sort of things so I'm 100% with you.